pkg/katautils/config.go

```diff
@@ -253,6 +254,21 @@ func (h hypervisor) machineAccelerators() string {
 	return machineAccelerators
 }
 
+func (h hypervisor) cpuFeatures() string {
+	var cpuFeatures string
+	features := strings.Split(h.CpuFeatures, ",")
+	featuresLen := len(features)
+	for i := 0; i < featuresLen; i++ {
+		if features[i] != "" {
+			cpuFeatures += strings.Trim(features[i], "\r\t\n ") + ","
```

Comment:
I think what you need here is `strings.TrimSpace`

Reply:
Ok